dev/test_all.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 npm --prefix=`dirname "$0"`/../verification/curator-service/api/ run-script test-silent
 npm --prefix=`dirname "$0"`/../data-serving/data-service/ run-script test-silent
-npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script test-silent
\ No newline at end of file
+npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script test-silent
+npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script cypress-ci-test
```

Comment:
s/cypress-ci-test/cypress-run ?

Reply:
Good catch. Also added comment that local stack is necessary for that command.